third_party/txt/src/txt/paragraph.h

```diff
@@ -17,52 +17,14 @@
 #ifndef LIB_TXT_SRC_PARAGRAPH_H_
```

Comment:
Should the interface definitions still live in the src/txt directory? I can see arguments for both leaving it or moving it one layer up.

Reply:
For now I'd like to keep this in `src/txt`.  Most of the contents of `src/txt` (`TextStyle`, `FontCollection`, etc.) will be used in both the Minikin and SkShaper based code paths.

If the code layout becomes too confusing, then in a later change we can move the Minikin-specific classes (`ParagraphTxt` and `ParagraphBuilderTxt`, plus a few support classes like `PaintRecord`) out of `src/txt`